ccmlib/scylla_node.py

```diff
@@ -115,6 +115,13 @@ def has_jmx(self):
 
         return self._has_jmx
 
+    @property
+    def scylla_yaml(self) -> Dict[str, Any]:
+        return yaml.load(Path(self.get_conf_dir()) / common.SCYLLA_CONF)
+    @property
```

Comment:
could you please add an empty line here?

Reply:
FWIW, we are encoding `10000` in dtest as well. i will fix them as well in piecemeal.

Reply:
> FWIW, we are encoding `10000` in dtest as well. i will fix them as well piecemeal.

And also in SCT, and other places.

I just noticed it in java-driver, that probably does it for no actual reason, historical reason let's call it